02_calculator/calculator.rb

```diff
@@ -0,0 +1,51 @@
+def add(x, y)
+  x+y
+end
+
+def subtract(x, y)
+  x-y
+end
+
+def sum(a = Array)
+  sumVal = 0;
+  for i in 0..a.size-1
+    sumVal += a[i]
```

Comment:
Ruby has a standard syle guide (as all the programming languages) https://github.com/rubocop/ruby-style-guide#snake-case-symbols-methods-vars